package.json

```
@@ -8,16 +8,21 @@
     "eslint-plugin"
   ],
   "author": "Dylan Praul",
+  "contributors": [],
   "main": "lib/index.js",
+  "bugs": "https://github.com/fintechstudios/eslint-plugin-chai-as-promised/issues",
+  "repository": "https://github.com/fintechstudios/eslint-plugin-chai-as-promised",
+  "homepage": "https://github.com/fintechstudios/eslint-plugin-chai-as-promised",
   "scripts": {
     "test": "mocha",
+    "lint": "eslint .",
     "preversion": "npm run test -- --single-run",
     "postversion": "git push --follow-tags && npm publish --access public"
   },
   "dependencies": {},
   "devDependencies": {
-    "eslint": "^3.9.1",
-    "mocha": "^3.5.3"
+    "eslint": "^6.8.0",
+    "mocha": "^7.0.1"
   },
   "engines": {
     "node": ">=0.10.0"
```

mocha 7 and eslint 6 require node versions over `^8.10.0`, but this is fine - we can do a major version bump. 

```suggestion
    "node": ">=8.10.0"
```